src/pages/ServerError.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Page from '../components/Page';
+
+import config from '../config';
+import { TitlePageHeader } from '../components/Header';
+import ErrorContent from '../components/ErrorContent';
+
+function ServerError() {
+  return (
+    <Page>
+      <TitlePageHeader dominion={config} profile={{}} />
+      <ErrorContent
+        title="500 - Internal Server Error"
+        description="We are having some trouble processing your request.
+              We have logged the error and will investigate. You can try again or if the issues persisit,
```

Comment:
1. `You can try again *later* and if...` or `Please try again *later* and if ...`,
2. Link `contact us` to contact page.